contribute/style-guides/documentation-style-guide.md

```diff
@@ -110,6 +110,8 @@ If that is the case, then use the following format for headings:
 - Assume all graphics will be exclusively viewed on the web.
 - Maximum image size is 3840px X 2160px.
 - Screenshots should be readable, but not too large.
+- _Do not_ use image shortcodes. Follow the guidance in the [Grafana markdown guide](https://github.com/grafana/grafana/blob/master/contribute/style-guides/documentation-markdown-guide.md#images).
+- Markdown image links are preferred. Only use the HTML image links if you need to style the image in ways unsupported in markdown.
```

Comment:
Markdown image links are preferred. Only use the HTML image links if you need to style the image in ways unsupported in markdown.

--> 

Markdown image links are preferred. Only use the HTML image links if you need to style the image in ways unsupported in Markdown.